mods/d2k/chrome/ingame-player.yaml

```diff
@@ -109,7 +109,7 @@ Container@PLAYER_WIDGETS:
 					DisableKeyRepeat: true
 					DisableKeySound: true
 					TooltipText: Deploy
-					TooltipDesc: Selected units will perform their default deploy activity\n - MCVs will unpack into a Construction Yard\n - Thumpers will start or stop attracting worms\n\nActs immediately on selected units.
+					TooltipDesc: Selected units will perform their default deploy activity\n - MCVs will unpack into a Construction Yard\n - Thumpers will start or stop attracting worms\n - Devastators will take damage over time and cause a huge explosion upon destruction\n\nActs immediately on selected units.
```

Comment:
`Devastators will take damage over time and cause a huge explosion upon destruction`
should be updated (e.g. `Devastators will be immobilized and cause a huge explosion after a delay`).

Reply:
Both of these lines are way too long for the tooltip.

Reply:
"Devastators will become immobilized and explode" would be a better length.

Reply:
Updated the tooltip.